cmd/util/ledger/migrations/storage_v6.go

```diff
@@ -1130,7 +1047,7 @@ func (c *ValueConverter) VisitArrayValue(_ *oldInter.Interpreter, value *oldInte
 
```

Comment:
Re: `if newElement != nil` above: Is there a chance that the migration will produce an array where some elements are `nil`?

Reply:
This is the case where, if some element is not migrated due to some error (missing type/ invalid type etc.), then we don't abort, but rather log the error and continues. In that case, migrated value (i.e: newElement) is nil.

Reply:
I see. We definitely make sure there are no such conversion errors reported after the migration runs. cc @Kay-Zee